### PR TITLE
examples/minikube: remove unused cilium-ds.yaml

### PR DESCRIPTION
**Pull request by aanm**
Since this file is not used in our documentation we can remove it to
avoid confusion to our users.

Signed-off-by: André Martins <andre@cilium.io>

---
This change is [Reviewable](https://reviewable.io/reviews/cilium/cilium/4351)

### CODE REVIEW

**Comment by aanm** [comment]
test-me-please

**Comment by aanm** [comment]
https://jenkins.cilium.io/job/Cilium-PR-Ginkgo-Tests-Validated/3782/testReport/junit/k8s-1/10/K8sValidatedChaosTest_Endpoint_can_still_connect_while_Cilium_is_not_running/

**Comment by aanm** [comment]
test-me-please

File: examples/minikube/README.rst [hunk 1]
```diff
@@ -1,26 +1,4 @@
...
-	$ minikube start --network-plugin=cni --bootstrapper=localkube --feature-gates=CustomResourceValidation=true
-        [...]
-	$ kubectl create -f cilium-ds.yaml
-
-Please see full `Getting Started on Kubernetes`_  guide for additional details
-on how to get started on Kubernetes including details on how to import
-Kubernetes NetworkPolicies.
-
-.. _Getting Started on Kubernetes: http://cilium.readthedocs.io/en/doc-1.0/kubernetes/
+https://cilium.readthedocs.io/en/latest/gettingstarted/minikube
```
Comment (ianvernon) [inline examples/minikube/README.rst:4]: Is 'latest' ideal here? What if this is accessed in some tagged version of Cilium?

Comment (aanm) [reply examples/minikube/README.rst:4]: This is for the latest documentation. The 1.1 should point to the 1.1

**Comment by aanm** [comment]
test-me-please

**Comment by aanm** [comment]
test-me-please

**Comment by aanm** [comment]
https://jenkins.cilium.io/job/Cilium-PR-Ginkgo-Tests-Validated/3842/

**Comment by aanm** [comment]
test-me-please

**Comment by aanm** [comment]
https://jenkins.cilium.io/job/Cilium-PR-Ginkgo-Tests-Validated/3857/

**Comment by aanm** [comment]
test-me-please

**Comment by tgraf** [comment]
Checked whether there were any meaningful references to "minikube/cilium-ds.yaml" and didn't find anything so: this is fine.

**Pull request merged**